pallets/external-agents/src/lib.rs

```diff
@@ -80,6 +81,11 @@ decl_storage! {
             map hasher(blake2_128_concat) Ticker
                 => AGId;
 
+        /// Maps an agent (`IdentityId`) to all all `Ticker`s they belong to, if any.
+        pub AgentOf get(fn agent_of):
+            map hasher(blake2_128_concat) IdentityId
+                => Vec<Ticker>;
```

Comment:
You're introducing an unbounded-ness attack vector into EAs here. Let's use a double map instead.